profiles/vee339.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/81091817/214922721-3a408d17-fc53-4278-b75e-a529663bcc97.png",
```

Comment:
I recommend more of an image like this. What do you think? 😃
P.S.: the badges seem to be failing due to an issue with the repository, I think (not being caused by the way you set it up).

![image](https://user-images.githubusercontent.com/51391473/214986098-f33de8e9-8e40-4d03-a869-0369edd0c915.png)